macros/provider_test.go

```diff
@@ -0,0 +1,38 @@
+package macros
+
+import (
+	"testing"
+
+	"github.com/prebid/prebid-server/exchange/entities"
+)
+
+func Test_macroProvider_GetMacro(t *testing.T) {
```

Comment:
**Nitpick:** Please rename this to `TestGetMacro`. In general, please use camel case for all test function names as per Go convention

Reply:
Done